cmd/minishift/cmd/config/config.go

```diff
@@ -100,17 +100,18 @@ var (
 	CacheImages  = createConfigSetting("cache-images", SetSlice, nil, nil, true)
 
 	// Pre-flight checks (before start)
-	SkipCheckKVMDriver    = createConfigSetting("skip-check-kvm-driver", SetBool, nil, nil, true)
-	WarnCheckKVMDriver    = createConfigSetting("warn-check-kvm-driver", SetBool, nil, nil, true)
-	SkipCheckXHyveDriver  = createConfigSetting("skip-check-xhyve-driver", SetBool, nil, nil, true)
-	WarnCheckXHyveDriver  = createConfigSetting("warn-check-xhyve-driver", SetBool, nil, nil, true)
-	SkipCheckHyperVDriver = createConfigSetting("skip-check-hyperv-driver", SetBool, nil, nil, true)
-	WarnCheckHyperVDriver = createConfigSetting("warn-check-hyperv-driver", SetBool, nil, nil, true)
-	SkipCheckIsoUrl       = createConfigSetting("skip-check-iso-url", SetBool, nil, nil, true)
-	WarnCheckIsoUrl       = createConfigSetting("warn-check-iso-url", SetBool, nil, nil, true)
-	SkipCheckVMDriver     = createConfigSetting("skip-check-vm-driver", SetBool, nil, nil, true)
-	WarnCheckVMDriver     = createConfigSetting("warn-check-vm-driver", SetBool, nil, nil, true)
-
+	SkipCheckKVMDriver     = createConfigSetting("skip-check-kvm-driver", SetBool, nil, nil, true)
+	WarnCheckKVMDriver     = createConfigSetting("warn-check-kvm-driver", SetBool, nil, nil, true)
+	SkipCheckXHyveDriver   = createConfigSetting("skip-check-xhyve-driver", SetBool, nil, nil, true)
+	WarnCheckXHyveDriver   = createConfigSetting("warn-check-xhyve-driver", SetBool, nil, nil, true)
+	SkipCheckHyperVDriver  = createConfigSetting("skip-check-hyperv-driver", SetBool, nil, nil, true)
+	WarnCheckHyperVDriver  = createConfigSetting("warn-check-hyperv-driver", SetBool, nil, nil, true)
+	SkipCheckIsoUrl        = createConfigSetting("skip-check-iso-url", SetBool, nil, nil, true)
+	WarnCheckIsoUrl        = createConfigSetting("warn-check-iso-url", SetBool, nil, nil, true)
+	SkipCheckVMDriver      = createConfigSetting("skip-check-vm-driver", SetBool, nil, nil, true)
+	WarnCheckVMDriver      = createConfigSetting("warn-check-vm-driver", SetBool, nil, nil, true)
+	SkipCheckVBoxInstalled = createConfigSetting("skip-check-vbox-installed", SetBool, nil, nil, true)
```

Comment:
So really you just add two lines here, right? The other changes are due to adding an additional whitespace for alignment, or?

Reply:
Yes, just added the last two lines, VSCode aligns and hence the other changes.

Reply:
ok